userspace/falco/app/options.cpp

```diff
@@ -147,6 +149,19 @@ bool options::parse(int argc, char **argv, std::string &errstr)
 
 	list_fields = m_cmdline_parsed.count("list") > 0 ? true : false;
 
+	int open_modes = 0;
+	open_modes += !trace_filename.empty();
```

Comment:
it seems like a joke that every engine is opened in a different way but that's the sad truth :joy: maybe one day we will us an enum